### DISCUSSION TITLE
Cloudy render

### DISCUSSION THREAD

**Discussion by seaprogrammer** [NVlabs/instant-ngp#791] [Q&A]
Whenever I've tried, I end up with a pretty cloudy/splotchy render view.

![Screenshot_20220718_205953](https://user-images.githubusercontent.com/54549366/179661725-6a9b4a7e-4525-46db-9c87-21d634edd3c4.png)

The images look like:
![Screenshot_20220718_210329](https://user-images.githubusercontent.com/54549366/179661872-5f621d4b-8c08-4a08-83f1-c237ab714288.png)

![Screenshot_20220718_210404](https://user-images.githubusercontent.com/54549366/179661933-41806dfd-d659-4ee6-acac-dbb48ac69fe0.png)

![Screenshot_20220718_210430](https://user-images.githubusercontent.com/54549366/179661972-31bc3af2-8500-4c39-97c2-166222c8f0bf.png)

**Comment by Tom94** [comment]
Hi there, make sure to carefully read the [NeRF dataset preparation tips](https://github.com/NVlabs/instant-ngp/blob/master/docs/nerf_dataset_tips.md), which include instructions on what kind of images tend to yield the best results.

What likely causes problems in the specific images above is the shallow depth of field and small field of view. Cheers!
